webclient/js/elems/raflow.js

```diff
@@ -1987,19 +1973,151 @@ window.loadRABGInfoForm = function () {
+                    size: '150px'
+                },
+                {
+                    field: 'LastName',
+                    caption: 'Last Name',
+                    size: '150px'
+                },
+                {
+                    field: 'IsCompany',
+                    caption: 'Is Company',
+                    size: '100px',
+                    render: function (record){
+                        if(record.IsCompany){
+                            return '<i class="fas fa-check" title="IsCompany" aria-hidden="true"></i>';
+                        }else {
+                            return '<i class="fas fa-times" title="IsCompany" aria-hidden="true"></i>';
+                        }
+                    }
+                },
+                {
+                    field: 'CompanyName',
+                    caption: 'Company Name',
+                    size: '100px'
+                },
+                {
+                    field: 'IsPayor',
+                    caption: 'Is Payor',
+                    size: '100px',
+                    render: function (record){
+                        if(record.IsPayor){
+                            return '<i class="fas fa-check" title="IsPayor" aria-hidden="true"></i>';
+                        }
+                    }
+                },
+                {
+                    field: 'IsUser',
+                    caption: 'Is User',
+                    size: '100px',
+                    render: function (record){
+                        if(record.IsUser){
+                            return '<i class="fas fa-check" title="IsUser" aria-hidden="true"></i>';
+                        }
+                    }
+                },
+                {
+                    field: 'IsGuarantor',
+                    caption: 'Is Guarantor',
+                    size: '100px',
+                    render: function (record){
+                        if(record.IsGuarantor){
+                            return '<i class="fas fa-check" title="IsGuarantor" aria-hidden="true"></i>';
+                        }
+                    }
+                }
+            ]
+        });
+
     }
 
     // now load form in div
-    $('#ra-form #bginfo').w2render(w2ui.RABGInfoForm);
+    $('#ra-form #bginfo').w2render(w2ui.RABGInfoGrid);
 
-    // load the existing data in people component
+    var i = getRAFlowPartTypeIndex(app.raFlowPartTypes.people);
+    var data = app.raflow.data[app.raflow.activeFlowID][i].Data;
+
+    console.log(data);
+
+    var usersInfo = data.Users;
+    var payorsInfo = data.Payors;
+    var gurantorsInfo = data.Guarantors;
+
+    // Peopleinfo list      0           1           2
+    var peopleInfoList = [usersInfo, payorsInfo, gurantorsInfo];
+
+    var raBGInfoGridRecords = [];
+    for(i = 0; i < peopleInfoList.length; i++){
+        for(var j = 0; j < peopleInfoList[i].length; j++){
+
+            // Assign checks for peoplelists
+            switch (i){
```

Comment:
@sudip-aubergine Please suggest an alternative to it.

Reply:
I just found issue with this approach. I'm fixing it. Please don't merge for now.